JSDemos/Demos/Toolbar/Multiline/jQuery/index.js

```diff
@@ -0,0 +1,370 @@
+const toolbarDataDefault = [
+  {
+    location: 'before',
+    widget: 'dxButton',
+    options: {
+      icon: 'undo',
+      onClick() {
+        DevExpress.ui.notify('Undo button has been clicked!');
+      },
+    },
+  },
+  {
+    location: 'before',
+    widget: 'dxButton',
+    options: {
+      icon: 'redo',
+      onClick() {
+        DevExpress.ui.notify('Redo button has been clicked!');
+      },
+    },
+  },
+  {
+    cssClass: 'menu-separator-container',
+    locateInMenu: 'auto',
+    location: 'before',
+    template(itemData, itemIndex, element) {
+      $(element).addClass('separator');
+    },
+    menuItemTemplate(itemData, itemIndex, element) {
+      $(element).addClass('menu-separator');
+    },
+  },
+  {
+    location: 'before',
+    locateInMenu: 'auto',
+    widget: 'dxDropDownButton',
+    options: {
+      displayExpr: 'text',
+      keyExpr: 'size',
+      useSelectMode: true,
+      items: fontSizes,
+      selectedItemKey: 14,
+      itemTemplate(itemData) {
+        return $('<div>')
+          .text(itemData.text)
+          .css('font-size', `${itemData.size}px`);
+      },
+    },
+  },
+  {
+    location: 'before',
+    locateInMenu: 'auto',
+    widget: 'dxDropDownButton',
+    options: {
+      icon: 'indent',
+      displayExpr: 'text',
+      keyExpr: 'lineHeight',
+      useSelectMode: true,
+      items: lineHeights,
+      selectedItemKey: 1.35,
+      onSelectionChanged(e) {
+        $('#text').css('line-height', e.item.lineHeight);
+      },
+    },
+  },
+  {
+    cssClass: 'menu-separator-container',
+    locateInMenu: 'auto',
+    location: 'before',
+    template(itemData, itemIndex, element) {
+      $(element).addClass('separator');
```

Comment:
Let's do more consistent with other approaches and add additional `div` element with correspond class $("<div>").addClass('').appendTo(element)

Reply:
or element.append($('div').addClass(''))

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/9b1f7460ae9d90f96eef54b91e00e769c2a65696